dadjokes-onchain/next/src/hooks/useJokes.js

```diff
@@ -0,0 +1,34 @@
+"use client";
```

Comment:
nit: is this a true react hook since it returns only jokes array and doesnt return a state updater? should it be in a helpers.js file instead outside the hooks dir?

Reply:
My understanding is that it's a hook b/c it has useState and useEffect, it's starts with an empty dependency array and ensures that the fetching logic runs once when the component that uses this hook mounts.